src/types/Error.ts

```diff
@@ -0,0 +1,7 @@
+export enum Error {
+  LoadTodosError = 'Unable to load todos',
+  EmptyTitleError = 'Title should not be empty',
+  AddTodoError = 'Unable to add a todo',
+  DeleteTodoError = 'Unable to delete a todo',
+  UpdateTodoError = 'Unable to update a todo',
```

Comment:
i don`t think it is necessary to use 'Error' in names cause you have already declared that it is enum Error, so it will be Error.LoadTodosError which is duplicating (i guess)